### ISSUE TITLE
Support of multiple concurrency settings on same worker

### ISSUE THREAD

**Issue by ziaulrehman40** [ixti/sidekiq-throttled#55]
Hi there,

I want to set concurrency of my job as such, only 10 concurrent jobs of specific class should run at a time. This is simple with: `sidekiq_throttle(concurrency: { limit: 10 })`.

But i want to add an additional check, which is that only 1 job of a key_suffic, (a specific parameter value) should run at a time. For example if my job was receiving two arguments, `user_id` and `doctor_id` i only want 10 jobs to run at max at a time, but also only one job per `doctor_id` to run in parallel.

How do i achieve this.
Hopefully, as k is clear. If it is not let me know i will rephrase.

BTW, thanks for the great gem. It really works. 👍

**Comment by ixti** [comment]
Sorry for not replying fast enough, it's not possible atm, but I have plans on supporting this in future.

**Comment by ixti** [comment]
Closing as duplicate of #17 (yes it's about threshold, but these are closely related to each other)
